### PR TITLE
App layout and folder structure setup

### PR DESCRIPTION
**Pull request by rtonata88**
In this PR I have setup the basic app layout as well as the folder structure following the ducks pattern.

### CODE REVIEW

File: src/components/BookItem.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const BookItem = () => (
+  <div className="bookItem">
+    <span>Book name</span>
+  </div>
```
Comment (D4nielJ) [inline src/components/BookItem.js]: - [x] Let's add a _Remove_ button to the individual Book. It should not have any functionality yet.

> Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet)

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Navbar from './components/Navbar';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Navbar />
```
Comment (D4nielJ) [inline src/index.js:11]: [OPTIONAL] I think it makes more sense to place the Navbar component inside of the App component as it makes part of the UI from the App. This is a little subjective though. Just a reminder that writing semantic code it's important to keep codebases understandable and organized.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @rtonata88, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: 
- Ducks Pattern folder structured is layout correctly!
- You have customized the README file to keep the repo professional. :clap: :fireworks: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookItem.js (hunk 1 above)
Comment (rtonata88) [reply src/components/BookItem.js]: This is done

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @rtonata88🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**